docs/userGuide/siteJsonFile.md

```diff
@@ -19,6 +19,7 @@ Here is a typical `site.json` file:
   "baseUrl": "/myproduct",
   "faviconPath": "myfavicon.png",
   "titlePrefix": "FooBar Dev Docs",
+  "titleSuffix": "",
```

Comment:
```suggestion
  "titleSuffix": "FooBar",
```

Just a suggestion: I think having an example string rather than an empty string will make it slightly better :)